policy.json

```diff
@@ -0,0 +1,22 @@
+{
+  "instance_min_count": 3,
+  "instance_max_count": 50,
+  "scaling_rules": [
+    {
+      "metric_type": "responsetime",
+      "breach_duration_secs": 60,
+      "threshold": 100,
+      "operator": "<",
+      "cool_down_secs": 120,
```

Comment:
This was a little arbitrary on my part, but is a hunch that once instance might make a big difference and we'd want to wait for a bit before trying again.

As a result we get a less snappy, but less jumpy form of gradual scaling?
What does everyone think?